### PR TITLE
BAU: Add Admin Postgres

### PR DESCRIPTION
**Pull request by amberstarlight**
# Pull Request

## What?

I have:

- Added a Postgres instance for the Admin app

## Why?

I am doing this because:

- We need a separate one for this service

### CODE REVIEW

**Comment by willfish** [comment]
We'll want to add one for production :raised_hands:

File: environments/development/common/rds.tf [hunk 1]
```diff
@@ -25,6 +26,35 @@ module "postgres" {
...
 }
 
+# Admin Postgres
+module "postgres_admin" {
+  source = "../../common/rds"
+
+  environment    = var.environment
+  name           = "PostgresAdmin"
+  engine         = "postgres"
+  engine_version = "13.11"
```
Comment (shjohnson) [inline environments/development/common/rds.tf:36]: Just question but is this the version we are currently using?

Comment (amberstarlight) [reply environments/development/common/rds.tf:36]: As far as I know. @willfish said it was fine 😇

**Comment by amberstarlight** [comment]
Renamed the branch so it wasn't confusing, so this auto-closed. Will open a new PR

File: environments/development/common/rds.tf (hunk 1 above)
Comment (shjohnson) [reply environments/development/common/rds.tf:36]: realise its just a small minor difference though

**Pull request closed without merging**